src/nvim/ui_compositor.c

```diff
@@ -0,0 +1,445 @@
+// This is an open source non-commercial project. Dear PVS-Studio, please check
+// it. PVS-Studio Static Code Analyzer for C, C++ and C#: http://www.viva64.com
+
+// UI wrapper that sends requests to the UI thread.
+// Used by the built-in TUI and libnvim-based UIs.
```

Comment:
This looks like a comment left over from ui_bridge. Is it relevant for ui_compositor?

Reply:
Still https://github.com/neovim/neovim/pull/9530 changes... But yes, must be leftover from initial copypasta.